app/assets/stylesheets/dm/pages/_partials/_header.scss

```diff
@@ -187,4 +187,23 @@ header {
       }
     }
   }
+
+  @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```

Comment:
nit: could we nest this within the `@media screen and (min-width: 64rem)` on `line 139` so we don't have to duplicate the media query?

Reply:
I believe I tried that, but it didn't work as intended.

Reply:
ooo lol css is hard... thanks, josh! please ignore this comment then ^__^